db/migrate/20190510161837_create_users.rb

```diff
@@ -0,0 +1,8 @@
+class CreateUsers < ActiveRecord::Migration[5.2]
+  def change
+    create_table :users do |t|
+      t.string :username, limit: 50, format: {with: /^[a-zA-Z]+$/}, uniqueless: false, null: false
```

Comment:
You don't need to create a new migration just to add `username` you can add it in the devise migration